state.js

```diff
@@ -145,14 +136,12 @@ createMachine({
                         },
                         onDone: [
                           {
-                            target:
-                              "#app.registration.logged_in.manage_tab.deletingApp.successDelete",
+                            target: "successDelete",
```

Comment:
Do we need to define it as a class like `. successDelete`?
same for the rest of the targets as well.

Reply:
So, this change does not really change they way the states have to be called. This was automatically formatted by xState where just writing successDelete here was enough. In this example, to call this state in css, I think it would still be `[data-state~='app.registration.logged_in.manage_tab.deletingApp.successDelete'] { css here }`

this string would be in the data state inside the body, which you can select. Here is an example of what the body looks like:

![Screenshot 2022-04-08 at 09 28 39](https://user-images.githubusercontent.com/80688204/162386373-37ff2973-0354-4111-8449-9a70cb9fee35.png)

For example, if you want to do a css implementation when the user is logged out. You select the `registration.logged_out` data state. You can do this by writing in CSS `[data-state~="registration.logged_out"] { 
css here }`.

Reply:
Thanks for the explanation @hubert-deriv 🙏🏼